### DISCUSSION TITLE
Road pavement surfaces

### DISCUSSION THREAD

**Discussion by iiv42** [Universite-Gustave-Eiffel/NoiseModelling#449] [Q&A]
Which would be the PVMT code for a dense asphalt concrete 0/11? AC11

Reference CNOSSOS: 0

thanks!!

**Comment by nicolas-f** [comment]
Hi,

Maybe it is **NL01**
You can check the parameters here:
https://github.com/Ifsttar/NoiseModelling/blob/cnossos/noisemodelling-emission/src/main/resources/org/noise_planet/noisemodelling/emission/coefficients_Road_Cnossos_2020.json#L341

**Reply by iiv42** [reply]
Thanks! but according to that NL01 is a 1 layer Porous Asphalt. 
Maybe the one named "DEF"? "without correction - Reference road surface"
